client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -11,19 +10,10 @@ import './style.scss';
 
 export default function SitesOverview(): ReactElement {
 	const translate = useTranslate();
-	const searchParam = new URLSearchParams( window.location.search ).get( 's' );
 
-	const [ searchQuery, setSearchQuery ] = useState( searchParam );
-	const { data, isError, isFetching } = useFetchDashboardSites( searchQuery );
+	const { search, currentPage } = useContext( SitesOverviewContext );
 
-	const handleSearch = ( query: string | null ) => {
-		setSearchQuery( query );
-	};
-	const dispatch = useDispatch();
-
-	useEffect( () => {
-		dispatch( recordTracksEvent( 'calypso_jetpack_agency_dashboard_visit' ) );
-	}, [ dispatch ] );
```

Comment:
Do we really want to remove the tracking of this event here? I can't see how it will be tracked otherwise.